src/components/FieldRegion/CreateFieldRegion.tsx

```diff
@@ -0,0 +1,57 @@
+import { useMutation } from '@apollo/client';
+import { Except } from 'type-fest';
+import { CreateFieldRegionInput, FieldZone, User } from '~/api/schema.graphql';
+import { DisplayFieldRegionFragment as FieldRegion } from '~/common';
+import { DialogForm, DialogFormProps } from '../Dialog/DialogForm';
+import { SubmitError, TextField } from '../form';
+import { FieldZoneField } from '../form/Lookup/FieldZone';
+import { UserField } from '../form/Lookup/User/UserField';
+import { CreateFieldRegionDocument } from './CreateFieldRegion.graphql';
+
+/*
+  CreateFieldRegionInput doesn't properly reflect the input from the fields, therefore the need for FieldRegionInput
+*/
+export interface FieldRegionInput {
+  fieldRegion: { name: string };
+  fieldZone: Partial<FieldZone>;
+  director: Partial<User>;
+}
```

Comment:
Take a look here:
https://github.com/SeedCompany/cord-field/blob/884a41d348a8594e2c6cb509f348f507ef19f539/src/scenes/Locations/LocationForm/LocationForm.tsx#L32-L44

We should try to follow suit of the input shape. That would put `director` & `fieldZone` under the `fieldRegion` sub object.
Also we shouldn't pull those objects (`FieldZone`, `User`) from the schema file directly. Instead we should use the fragments that those fields require: `DisplayFieldZoneFragment`, `UserLookupItemFragment`